environment.py

```diff
@@ -362,11 +364,24 @@ def env() -> Mapping[str, Optional[str]]:
         # Typically only set for main deployments.
         'AZUL_ENABLE_MONITORING': '0',
 
-        # The default bundle UUID prefix to use for reindexing bundles in the DSS
-        # and for subscriptions to the DSS. If this variable is set to a non-empty
-        # string, only bundles whose UUID starts with the specified string will be
-        # indexed.
-        'AZUL_DSS_QUERY_PREFIX': '',
+        # Identifies the DSS repository endpoint and prefix to index.
+        # The syntax in EBNF is:
+        #
+        # source = 'https://', net_loc,
+        #          [ '/', path ],
+        #          ':', [ UUID prefix ], '/', partition prefix length ;
+        #
+        # The non-terminals `net_loc` and `path` are defined as in
+        # https://datatracker.ietf.org/doc/html/rfc1808
+        #
+        # The EBNF syntax for `UUID prefix` and `partition prefix length` is
+        # defined along with the documentation of `AZUL_CATALOGS` within this file
+        #
+        # Examples:
+        #
+        # https://dss.data.humancellatlas.org/v1:
```

Comment:
Even the first example doesn't match the grammar.

Reply:
https://github.com/DataBiosphere/azul/pull/3463#discussion_r769176108